sirepo/server.py

```diff
@@ -294,12 +294,24 @@ def app_find_by_name(simulation_type, application_mode, simulation_name):
 def app_import_file(simulation_type):
     f = flask.request.files['file']
     try:
-        srwblParam, appParam, el, pp = sirepo.sirepo_parser.sirepo_parser(f.read())
-        args = sirepo.sirepo_parser.list2dict(srwblParam)
-        v = sirepo.sirepo_parser.Struct(**args)
-        args = sirepo.sirepo_parser.list2dict(appParam)
-        app = sirepo.sirepo_parser.Struct(**args)
-        data = sirepo.sirepo_parser.parsed_dict(v, app, el, pp)
+        from werkzeug import secure_filename
+        filename = secure_filename(f.filename)
+        imported_file_path = os.path.join('./', filename)
```

Comment:
We shouldn't store relative paths. Look at 
[app_upload_file](https://github.com/radiasoft/sirepo/blob/master/sirepo/server.py#L552). 
This code needs to be factored out. Possibly need to have an "imports" directory to avoid collisions. Use py.local and pkio.mkdir_parent_only() so that you know the directory exists.

Also, we don't want to grab just any old name. They could be rather long. We should generate the saved name, and store the actual name in a comment in the upload or the generated json.